src/initial-submission/components/DetailsForm.tsx

```diff
@@ -21,6 +21,19 @@ interface Props {
     ButtonComponent?: (props: { saveFunction?: Function }) => JSX.Element;
 }
 
+const useSuggestions = (values: ManuscriptDetails, suggestions: Array<Suggestion>): ManuscriptDetails => {
+    const detail = (values ? values : {}) as ManuscriptDetails;
+    const titleIsBlank = !detail.title || detail.title === '';
+    const suggestedTitle = suggestions.reduce(
```

Comment:
could this be done as 
```js
suggestions.find(suggestion => suggestion.fieldName === 'title');
```

Reply:
Ooo nearly ... need to add ".value" on the end ... thanks